index.html

```diff
@@ -30,7 +32,7 @@ <h2>A small tool for generating and downloading Powercord Manifest Files</h2>
                 <input type="button" value="Plugins" class="selectItem selected" data-type="plugins" />
                 <input type="button" value="Themes" class="selectItem" data-type="themes" />
             </div>
-            <div class="generator selected" data-type="plugins" data-fileName="manifest.json">
```

Comment:
this is for the download part and should not be changed as manifest files are called like that

Reply:
Changed back to original.